index.html

```diff
@@ -98,25 +102,37 @@
   <script>
     const symbols = [
       "©", "®", "™", "“", "”", "£", "…", "½", "¼", "∞", "é", "á", "à", "ç", "€",
-      "¥", "₩", "¢", "¤", "œ", "Œ", "æ", "Æ", "✔", "°", "²", "³", "‽"
+      "¥", "₩", "¢", "¤", "œ", "Œ", "æ", "Æ", "✔", "°", "²", "³", "‽", { display: "ZWSP", character: "\u{200B}" }
     ];
 
     document.addEventListener("DOMContentLoaded", () => {
       const parent = document.querySelector(".symbols");
       for (const symbol of symbols) {
         const elem = document.createElement("div");
         elem.classList = "symbol";
-        elem.textContent = symbol;
+
+        let display, character;
+
+        if (typeof symbol === "string") {
+          display = symbol;
+          character = symbol;
+        } else {
+          display = symbol["display"];
+          character = symbol["character"];
+          elem.classList += " symbol-long";
```

Comment:
You can do `elem.classList.add('symbol-long')` to avoid the `+=`.  You can also use `.remove()` in the `setTimeout` (though I question why that's necessary).